src/playground/sample-data-ingestion/generator.py

```diff
@@ -0,0 +1,187 @@
+from faker import Faker
```

Comment:
what's the version of Faker ? could you please define your dependency version ? thanks